CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+
+3.0.2 (xx.06.2020)
+------------------
+
+* Added support for Wagtail 2.9 (no code changes necessary).
+* Added compatibility with Django Sites Framework.
+* Modify tox configuration file to ensure Django 1.11/2.1/3.0 are in dependencies.
+* Modify Travis CI configuration file to test against Django 2.1, Wagtail 2.9.
```

Comment:
Could we perhaps remove these two lines? I would classify these changes as part of adding support for 2.9. Also, please add your name in brackets next to the other items to credit yourself :)